.github/workflows/cd.yaml

```
@@ -0,0 +1,62 @@
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.CD_PAT }}
+          submodules: recursive
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
```

actionlint (1.7.4)

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)